### PR TITLE
Add win streak to leaderboard

### PR DESCRIPTION
**Pull request by jbarr21**
- Prints out winning streak info in leaderboard if a minimum threshold is met
- `get_win_streak` is a separate func which can be reused for output to winner after a confirm msg
- Threshold is easily adjustable

Example output:

```
Name            ELO    Wins    Losses  Streak
------------  -----  ------  --------  ----------------
@kevin   1500       6         1  (won 5 in a row)
@chichi   1500       0         1
@james   1452       1         5
```

### CODE REVIEW

File: elobot.py [hunk 1]
```diff
@@ -15,6 +15,8 @@
 LEADERBOARD_REGEX = re.compile('Print leaderboard', re.IGNORECASE)
 UNCONFIRMED_REGEX = re.compile('Print unconfirmed', re.IGNORECASE)
 
+MIN_STREAK_LEN = 3
```
Comment (mburst) [inline elobot.py]: Can we move this to a config var? That way there are no hard coded constants

File: elobot.py [hunk 2]
```diff
@@ -143,6 +147,18 @@ def print_unconfirmed(self):
     def is_bot(self, user_id):
         return self.slack_client.api_call('users.info', user=user_id)['user']['is_bot']
 
+    def get_win_streak(self, player_slack_id):
+        win_streak = 0
+        matches = Match.select().where((player_slack_id == Match.winner) | (player_slack_id == Match.loser)).order_by(Match.played.desc())
+        for match in matches:
+            print 'match id: ' + str(match.id) + ', winner: ' + match.winner_id + ', and loser: ' + match.loser_id
```
Comment (mburst) [inline elobot.py]: Don't think you need this in here

**Comment by mburst** [comment]
One idea for this would be to make a column on the accounts table with a streak column as an integer. Then update it on wins and losses. That way you don't need to make a db query for each player. Not a huge deal though given the size of the db. Just something to think about.

Might also be cool to use W and L notation as other sports do for streaks:
image

That way you don't even need to worry about having a min streak length

File: elobot.py (hunk 1 above)
Comment (jbarr21) [inline elobot.py]: ah, yeah, will do. i started at a constant and forgot to move external

File: elobot.py (hunk 2 above)
Comment (jbarr21) [inline elobot.py]: good catch, will remove

**Comment by jbarr21** [comment]
@mburst all changes except for new streak DB column made, tested, and pushed.

i like the STRK, L10, PCT, W, and L idea, but didn't want to deal with schema migrations just yet. if you would rather see that solution instead of this, just let me know and I can close this PR

**Comment by mburst** [comment]
I'm good with this for now. We can work on a migration at a later time. Our DB isn't that big yet to worry about it

**Pull request merged**